### PR TITLE
Reimplementing for sww 0.2.0

### PR DESCRIPTION
**Pull request by delapuente**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,22 @@
-#DynamicCache
-
-A middleware layer for [ServiceWorkerWare](https://github.com/arcturus/serviceworkerware/) that will allow you to modify the content of a cache.
+# RawCache
+A middleware layer for [ServiceWorkerWare](https://github.com/gaia-components/serviceworkerware/) that will allow you to modify the content of a cache.
```
Comment (arcturus) [inline README.md]: I would like to add at the end:

... content of a cache via HTTP verbs.

wdyt?

File: README.md [hunk 2]
```diff
@@ -31,15 +27,10 @@ You'll find the following methods:
 
 Why there is no `get` method? The middleware will be returning the content during the `onFetch` phase, so any HTTP request will give you the content.
 
-
 ## Modifying the content
 All this functions are available if you execute them in the ServiceWorker scope. So you can call the previous methods to perform actions.
-But this middleware allows you to modify the content through any client that has access to the worker. The communication is done via
-`postMessage` (or [BroadcastChannel API](https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API) if available) with an specific protocol.
 
+But this middleware allows you to modify the content through any client that has access to the worker. The communication is done via `postMessage` (or [BroadcastChannel API](https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API) if available) with an specific protocol.
```
Comment (arcturus) [inline README.md]: This cache just uses http verbs, we removed the postMessage, or at least is in a different repo.

Don't know if we should mix them and allow both mechanism or just leave this at it is. Good news both methods (except the BroadcastChannel) will wake up the serviceworker.

File: README.md (hunk 1 above)
Comment (delapuente) [inline README.md]: cool

File: README.md (hunk 2 above)
Comment (delapuente) [inline README.md]: Lets leave this as is. Once we have the control url, to implement a wrapper is easy.

**Pull request merged**